lib/ZumoOta/src/WebServerCustom.cpp

```diff
@@ -69,9 +70,49 @@ WebServerCustom::~WebServerCustom()
 {
 }
 
+String WebServerCustom::listFiles(bool ishtml)
+{
+  String returnText = "";
+  LOG_DEBUG("Listing files stored on LittleFS");
+  File root = LittleFS.open("/");
+  File foundfile = root.openNextFile();
+  if (ishtml) {
+    returnText += "<table><tr><th align='left'>Name</th><th align='left'>Size</th></tr>";
+  }
+  while (foundfile) {
+    if (ishtml) {
+      returnText += "<tr align='left'><td>" + String(foundfile.name()) + "</td><td>" + humanReadableSize(foundfile.size()) + "</td></tr>";
+    } else {
+      returnText += "File: " + String(foundfile.name()) + "\n";
+    }
+    foundfile = root.openNextFile();
+  }
+  if (ishtml) {
+    returnText += "</table>";
+  }
+  root.close();
+  foundfile.close();
+  return returnText;
+}
+
+String WebServerCustom:: humanReadableSize(const size_t bytes)

```

Comment:
the flash size of the robot is I belive 32kb only, I strongly doubt to ever see hex files  beyond 100kb.

Reply:
It will be taken care of by the next commit.